README.md

```diff
@@ -87,6 +87,30 @@ days thence that Event will be processed.
 - Routine scheduling behaviors allow rules like run-once, schedule-once, last-only (cancel any existing Events when one is scheduled)
 - Task groups allow scheduling behavior to be applied to multiple Routines within a group
 
+# Config
+`UMEBOSHI_SERIALIZER`: Path to custom data serializer 
```

Comment:
@markhsu-ms You might want to add a note about if this setting is not set then the default will be used

Reply:
Good idea